lib/ecstatic/opts.js

```diff
@@ -90,6 +91,29 @@ module.exports = function (opts) {
     });
 
     [
+      'H',
+      'header',
+      'headers'
+    ].forEach(function (k) {
+      if (!isDeclared(k)) return;
+      if (Array.isArray(opts[k])) {
+        opts[k].forEach(setHeader);
```

Comment:
Interesting way to do this. I was expecting the array/string part to be special cased to the bin handler.

Not a bad thing.